composer.json

```diff
@@ -16,9 +16,9 @@
 		"wmde/email-address": "~1.0",
 		"wmde/euro": "~1.0",
 		"wmde/clock": "~1.0",
-		"wmde/fundraising-donations": "~9.1",
-		"wmde/fundraising-memberships": "~7.0",
-		"wmde/fundraising-payments": "~4.0",
+		"wmde/fundraising-donations": "dev-new-payment-domain",
+		"wmde/fundraising-memberships": "dev-change-create-membership-UC",
```

Comment:
branch was merged, this is [update-payments](https://github.com/wmde/fundraising-memberships/tree/update-payments` again, otherwise dependencies won't install